generators/client/templates/angular/src/test/javascript/jest.conf.js.ejs

```diff
@@ -13,7 +13,7 @@ module.exports = {
     },
     reporters: [
         'default',
-        [ 'jest-junit', { output: './<%= BUILD_DIR %>test-results/jest/TESTS-results.xml' } ]
+        [ 'jest-junit', { output: './<%= BUILD_DIR %>surefire-reports/TESTS-results-jest.xml' } ]
```

Comment:
@Tcharl You forget to change the path to 'test-results' here also

Reply:
Thanks! changed pushed